mlogger/config.py

```diff
@@ -5,19 +5,29 @@
 import time
 import sys
 import mlogger
+import warnings
 
 
 class Config(object):
-    def __init__(self, plotter=None, plot_title=None,
-                 get_general_info=True, get_git_info=False, **kwargs):
+    def __init__(self, plotter=None, plot_title=None, get_general_info=True, 
+                 get_git_info=False, visdom_plotter=None, summary_writer=None, **kwargs):
 
         object.__setattr__(self, '_state', {})
 
         if plotter is not None:
-            self.plot_on(plotter, plot_title)
-        else:
-            object.__setattr__(self, '_plotter', plotter)
-            object.__setattr__(self, '_plot_title', plot_title)
+            warnings.warn("use visdom_plotter instead of plotter", FutureWarning)
```

Comment:
Can you change this to `warnings.warn("Argument `plotter` is deprecated. Please use `visdom_plotter` instead.", DeprecationWarning)`

Reply:
Will change it now

Reply:
My bad about changing `FutureWarning` to `DeprecationWarning`, you were right https://docs.python.org/3/library/exceptions.html#FutureWarning